src/tvheadend/entity/Recording.h

```diff
@@ -172,6 +177,7 @@ namespace tvheadend
       void SetPriority(uint32_t priority) { m_priority = priority; }
 
     private:
+      uint32_t         m_enabled;
```

Comment:
This is how all the "HTSP" types in pvr.hts currently work. They store the htsp types internally, offer setters for htsp types and getters for Kodi types. So, all good imo, because in line with the rest. Okay?